### PR TITLE
Fix data analysis

### LINKED ISSUE #234: Data visualisation corrections
Data visualisation tab has some issues to be addressed:
- [ ] t-SNE plot button differs from other buttons.
- [ ] Data analysis runs on raw data. Give option to analise with both cases: raw and normalised.

### PR DESCRIPTION
**Pull request by EdAguilarB**
## Description
Corrects data analysis tab. Button to save plots is "Save Plot" for all the plots. Normalisation of features and perplexity added as user variables to the UI.

## Linked issues
- Closes #234 

## (Optional) Screenshots
image

### CODE REVIEW

File: biofefi/components/forms.py [hunk 1]
```diff
@@ -543,51 +544,107 @@ def pairplot_form(data, data_analysis_plot_dir, plot_opts):
 
 
 @st.experimental_fragment
-def tSNE_plot_form(data, random_state, data_analysis_plot_dir, plot_opts):
+def tSNE_plot_form(data, random_state, data_analysis_plot_dir, plot_opts, scaler=None):
 
     from sklearn.manifold import TSNE
-    from sklearn.preprocessing import StandardScaler
+    from sklearn.preprocessing import MinMaxScaler, StandardScaler
```
Comment (dcl10) [inline biofefi/components/forms.py:550]: How come you're doing the import inside the function instead of at the top of the file?

File: biofefi/components/forms.py [hunk 2]
```diff
@@ -543,51 +544,107 @@ def pairplot_form(data, data_analysis_plot_dir, plot_opts):
 
 
 @st.experimental_fragment
-def tSNE_plot_form(data, random_state, data_analysis_plot_dir, plot_opts):
+def tSNE_plot_form(data, random_state, data_analysis_plot_dir, plot_opts, scaler=None):
```
Comment (dcl10) [inline biofefi/components/forms.py:547]: Type annotation for `scaler` please

**Review by dcl10 — CHANGES_REQUESTED** [review]

**Review by dcl10 — APPROVED** [review]
